pkg/login/handle.go

```diff
@@ -134,6 +136,11 @@ func (h Handle) PostLogin(w http.ResponseWriter, r *http.Request) *Response {
 	}
 	copier.Copy(&response.User, dbUsers[0])
 
+	if !h.IsProdEnv {
```

Comment:
We probably don't need to make it this complex to return in body. I think all that is needed is to toggle the HttpOnly flag in the setTokenCookie function. (At least based on what I remember from a previous conversation we had when we first switched over to using this method)

HttpOnly = true in prod
HttpOnly = false in qa or local